frontend/src/sass/elements/_sidebar.scss

```diff
@@ -81,7 +81,7 @@
   margin-right: 14px;
   position: absolute;
   width: 296px;
-  top: 250px;
+  top: 0;
```

Comment:
I'm not 100% sure this is coming from here, but there now the sidebar  jumps when you scroll on the other pages that have sidebars .

Reply:
By jump do you mean the sidebar is not positioned near the top of the page?
<img width="698" alt="screen shot 2018-07-13 at 1 46 13 pm" src="https://user-images.githubusercontent.com/1178494/42705944-0dbaa98e-86a3-11e8-8d33-8ea5bc073210.png">

Reply:
Yeah, that on the learn more and xmas navigations, there is a scrollY limit that it hits and just goes down about 75-100px as you scroll down. I can't tell if that behavior is intentional.

Reply:
Yeah thats the current behavior on the deployed site. When I go through and try and use the same shared component for everything I can make it aligned to the top of the page in the same way the temp outfitter side bar is.

Reply:
I adjusted the CSS (and HTML markup) so that both implementations should appear and function exactly the same. The learn more pages should now have the sidebar content top aligned when it is in a fixed position rather than floating in the center of the page.

Reply:
👍